bacs/src/main/java/com/adyen/checkout/bacs/BacsDirectDebitConfiguration.kt

```diff
@@ -63,13 +69,23 @@ class BacsDirectDebitConfiguration private constructor(
             clientKey
         )
 
+        /**
+         * TODO docs
+         */
+        override fun setSubmitButtonEnabled(isSubmitButtonEnabled: Boolean): Builder {
```

Comment:
Should we call it `isSubmitButtonEnabled` or `isSubmitButtonVisible`? "enabled" sounds like it will be visible but greyed out when set to false imo